imagefiles/dockcross

```diff
@@ -180,12 +180,17 @@ fi
 # Now, finally, run the command in a container
 #
 tty -s && TTY_ARGS=-ti || TTY_ARGS=
-docker run $TTY_ARGS --rm \
+CONTAINER_NAME=dockcross
+docker run $TTY_ARGS --name $CONTAINER_NAME \
     -v $PWD:/work \
     $USER_IDS \
     $FINAL_ARGS \
     $FINAL_IMAGE "$@"
```

Comment:
We probably need

```
return_code=$?
```

here then,

```
exit $return_code
```

after the `docker rm` call to pass along the return code of the command that is executed.